app/src/main/java/de/tum/in/tumcampusapp/database/Converters.kt

```diff
@@ -9,10 +9,14 @@ import org.joda.time.DateTime
 
 class Converters {
     @TypeConverter
-    fun isoToDateTime(str: String): DateTime = DateTimeUtils.getDateTime(str)
+    fun isoToDateTime(str: String): DateTime? {
+        return if (str.isNotEmpty()) DateTimeUtils.getDateTime(str) else null
+    }
 
     @TypeConverter
-    fun fromDateTime(date: DateTime): String = DateTimeUtils.getDateTimeString(date)
+    fun fromDateTime(date: DateTime?): String {
+        return if (date != null) DateTimeUtils.getDateTimeString(date) else ""
+    }
 
```

Comment:
@kordianbruck is it ok if we change these converter methods to also accept null? or should we solve this within our class.

Reply:
This should be more correct:

```
@TypeConverter
    public static Long toLong(Date value) {
        return value == null ? null : value.getTime();
    }
```

That way you will also have `null`s in the db.

Reply:
@schulzalexander what about this?

Reply:
@kordianbruck sorry forgot that.
You mean like
```
@TypeConverter
    fun isoToDateTime(str: String?): DateTime? {
        return if (str == null) null else DateTimeUtils.getDateTime(str)
    }

    @TypeConverter
    fun fromDateTime(date: DateTime?): String? {
        return if (date == null) null else DateTimeUtils.getDateTimeString(date)
    }
```
?

Reply:
Looks good yeah. Make sure the Model fields are nullable as well.

Reply:
ok, yes the "end" property is nullable already.

Reply:
@kordianbruck pushed that change